object-store-client-common/src/main/scala/uk/gov/hmrc/objectstore/client/model/http/ObjectStoreWrite.scala

```diff
@@ -18,8 +18,12 @@ package uk.gov.hmrc.objectstore.client.model.http
 
 import uk.gov.hmrc.objectstore.client.model.Monad
 
+import scala.annotation.implicitNotFound
+
 case class Payload[CONTENT](length: Long, md5Hash: String, content: CONTENT)
 
+@implicitNotFound("""No implicits found for ObjectStoreContentWrite[${F}, ${CONTENT}, ${BODY}].
+If you are using a Source[ByteString, _], you may be missing an implicit Materializer""")
```

Comment:
It becomes quite a list, but if F uses Future, it cold be the missing implicit ExecutionContext too.

Reply:
I've added the ExecutionContext message on ObjectStoreRead which should be sufficient as F has to be the same.

Reply:
You wouldn't see the message from `ObjectStoreRead` if you just require a ObjectStoreContentWrite